tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1708,6 +1710,98 @@ public function theUserHasCreatedASpaceByDefaultUsingTheGraphApi(
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" copies (?:file|folder) "([^"]*)" to "([^"]*)" in space "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $fileSource
+	 * @param string $fileDestination
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 */
+	public function userCopiesFileWithinSpaceUsingTheWebDAVAPI(
+		string $user,
+		string $fileSource,
+		string $fileDestination,
+		string $spaceName
+	):void {
+		$space = $this->getSpaceByName($user, $spaceName);
+		$headers['Destination'] = $this->destinationHeaderValueWithSpaceName(
+			$user,
+			$fileDestination,
+			$spaceName
+		);
+
+		$fullUrl = $this->baseUrl . "/dav/spaces/" . $space['id'] . '/' . $fileSource;
```

Comment:
what if $fileSource is like `/newFile.txt`
your fullUrl will be like: `<some-base-url>/dav/spaces/<some-space-ic>//newFile.txt`